opentelemetry-otlp/README.md

```diff
@@ -38,7 +38,7 @@ First make sure you have a running version of the opentelemetry collector you
 want to send data to:
 
 ```shell
-$ docker run -p 55680:55680 otel/opentelemetry-collector-dev:latest
+$ docker run -p 4317:4317 otel/opentelemetry-collector-dev:latest
```

Comment:
The support for this port has been released in the Docker image: https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.16.0